core/test/common/FixedPoint.js

```diff
@@ -195,6 +293,17 @@ contract("FixedPoint", function(accounts) {
     assert(await didContractThrow(fixedPoint.wrapMixedDiv("1", "0")));
   });
 
+  it("Mixed division, with ceil", async function() {
+    const fixedPoint = await FixedPointTest.new();
+
+    // Normal mixed division case.
+    let quotient = await fixedPoint.wrapMixedDivCeil(web3.utils.toWei("150.3"), "3");
```

Comment:
What about dividing toWei(1) by 10 ** 76? I *think* normal mixed div will return 0, but divCeil will overflow? Could be totally wrong, and also we could ignore this case as well.

Reply:
Correct:
- Normal mixed div returns 0
- mixedDivCeil overflows

Reply:
Added a comment to `divCeil` and a test to `Mixed Division, with ceil` to indicate that large denominators can cause overflow